### DISCUSSION TITLE
Implement an easy-to-use Linux installer

### DISCUSSION THREAD

**Discussion by GBX9570** [CosmosOS/Cosmos#3091] [Ideas]
While it is overall very simple to install Cosmos on Linux, some may prefer to have a one click command. I have made a version myself, however, it only installs dotnet on Debian due to the limited compatibility. It is very possible to install dotnet on other distros, however that is out of the scope of this due to limited testing and varying differences between distros. The script I have made supports the following distros:
Debian
Fedora
Arch
Void Linux
OpenSUSE
alpine Linux
Gentoo
NixOS
Solus
This does require more testing as I have not checked to see if it works at all apart from on void, debian and fedora. Before, and if, this is added to Cosmos source code, I recommend further testing.
[linux-cosmos-install.zip](https://github.com/user-attachments/files/17211731/linux-cosmos-install.zip)

**Comment by randomusert** [comment]
You could Make IT get The install script and then The script could install dotnet with it. Pretty simple actually

**Comment by ghost** [comment]
There is already a version for linux, for more information: https://cosmosos.github.io/articles/Installation/DevKit.html#linux
Also its easy you just need use the "make" command in Cosmos directory

**Reply by randomusert** [reply]
also for easy dotnet install use the install script
